packages/trpc/server/routers/viewer/dsync/teamGroupMapping/create.handler.ts

```diff
@@ -0,0 +1,30 @@
+import prisma from "@calcom/prisma";
+import type { TrpcSessionUser } from "@calcom/trpc/server/trpc";
+
+import { TRPCError } from "@trpc/server";
+
+import type { ZCreateInputSchema } from "./create.schema";
+
+type Options = {
+  ctx: {
+    user: NonNullable<TrpcSessionUser>;
+  };
+  input: ZCreateInputSchema;
+};
+
+export const createHandler = async ({ ctx, input }: Options) => {
+  if (!ctx.user.organizationId) {
```

Comment:
same for all other handlers below: 

Shouldn't we use   `const { message, access } = await canAccess(ctx.user, orgId);` here instead again? + also check if teamId belongs to org 

`authedOrgAdminProcedure` already checks that the user is part of an org